gapic/samplegen/samplegen.py

```diff
@@ -86,48 +92,147 @@ class InvalidRequestSetup(SampleError):
     pass
 
 
-class DuplicateInputParameter(SampleError):
-    pass
+class Validator:
+    """
+    Class that validates samples.
```

Comment:
nit: (cf https://github.com/google/styleguide/blob/gh-pages/pyguide.md#381-docstrings)
```
"""Class that validates samples.

Contains...
...
"""
```

See examples in